stix_shifter_modules/qradar/configuration/qradar_config.json

```diff
@@ -4,7 +4,8 @@
             "default": "QRadar"
         },
         "host": {
-            "type": "text"
+            "type": "text",
+            "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$"
```

Comment:
this regex has to applied to all the hosts.

Reply:
you need to apply regex to name and description fields as well

Reply:
yes I was going to update all config hosts. why do we need to apply regex in name and description? name and description are base config. UI already restricts some characters. and also name and description is not used anywhere in stir-shifter.

Reply:
it is a security restriction.

Reply:
@yurii-klymenko I understand the security restriction. But do you know what special characters we should allow for name and description? I asked Luke he doesn't have anything validation for name and description. we need to be align with UI in this case. For simplicity, I can just put a regex that allows only string and integer but not sure that's sufficient.